### DISCUSSION TITLE
new beta tag

### DISCUSSION THREAD

**Discussion by teleshoes** [stepmania/stepmania#2185] [General]
it seems to me theres plenty of new stuff, and no particular roadmap or deadlines. why not tag HEAD and build some binaries?
(this is a real question; is there some reason why not?)

**Comment by Sudospective** [comment]
![Screenshot_20220302-115851](https://user-images.githubusercontent.com/11000739/156440011-90465472-f618-4ed1-a4e5-b75cb3b02492.png)
youre either talking about a different branch or hard clowning

**Comment by teleshoes** [comment]
the former. i hadnt seen a b3 tag, only the b2 that is widely used. 5_1_new is where the new stuff is if memory serves
